react/features/app-settings/components/AbstractAppSettings.js

```diff
@@ -304,8 +173,19 @@ export class AbstractAppSettings extends Component<Props, State> {
  * @returns {Object}
  */
 export function _mapStateToProps(state: Object) {
+    const _configDefaultURL = state['features/app'].app._getDefaultURL();
+    const _profile = getProfile(state);
+
+    console.log('ZB ::: New profile', JSON.stringify(_profile));
+
+    if (typeof _profile.defaultURL !== 'string') {
```

Comment:
why do we need this? why not leave it as undefined? right now it shows in "black" as if the user typed it, IMHO it should remain undefined.